bin/run-css-lint

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+const child_process = require( 'child_process' );
+
+var args = [ '**/*.scss'," --syntax scss"];
+if ( process.argv.length > 2 ) {
+	args = args.concat( process.argv.slice( 2 ) );
+} else {
+	args = args.concat( [ '.' ] );
+}
+
+const cssResults = child_process.spawnSync( 'stylelint', args );
```

Comment:
Were you able to get this script running? I'm getting this as the output of this command:  { [Error: spawnSync stylelint ENOENT]
     code: 'ENOENT',
     errno: 'ENOENT',
     syscall: 'spawnSync stylelint',
     path: 'stylelint',
     spawnargs: [ '**/*.scss', ' --syntax scss', '.' ] }